docs/faq.md

```diff
@@ -0,0 +1,27 @@
+# Frequently asked questions
+
+## Does it work well with transient data, e.g. new metrics constantly appearing and other metrics becoming inactive, such as those from docker containers ?
+
+For any metric, we only store data that is received, as well as an entry into the index.  So the overhead of a shortlived metric is low.
+
+Furthermore, we have optimizations for this use case:
+
+* Index filtering: when you request data, we exclude items from the result set that have not been updated in 24hours before the "from" of the request (as the data will be all null anyway)
+* Index pruning: if enabled, we delete series from the index if no data has been received in "max-stale". (but keep data until it expires, in case the same metric gets re-added). This is useful beecause the query editor does not send a time range. Note that this setting is applied to *all* metrics.
```

Comment:
`beecause` double `e`.
also: I'd say `"max-stale" time` just because it sounds a little more intuitive